.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,5 @@
+<component name="ProjectCodeStyleConfiguration">
+  <code_scheme name="Project" version="173">
+    <option name="ALIGN_MULTILINE_PARAMETERS_IN_CALLS" value="true" />
+  </code_scheme>
+</component>
```

Comment:
Newline is missing here. Not sure if this can cause problems but there is a warning here in the github web interface so safest to end the file with a new line i think